pages.pt-BR/common/maven.md

```diff
@@ -0,0 +1,36 @@
+# mvn
+
+> Ferramenta para a criação e gerenciamento de projetos Java.
+> Página Oficial: <https://maven.apache.org/>.
```

Comment:
Até agora nós usavamos "Homepage" antes dos links. Não devemos continuar assim até que a discussão em #3041 seja terminada?

Reply:
@schneiderl the discussion at #3041 is not about which term to use to indicate links, it's about whether or not core utils should have links to web-hosted man pages. This is a different matter.

Reply:
@mebeim is right, but indeed the discussion on what the label (when it's present) should be is still ongoing, as I noted [here](https://github.com/tldr-pages/tldr/pull/3030#issuecomment-495006814). That said, this PR shouldn't be held back while we make that decision.